docs/_posts/2018-02-22-units-of-measurement.md

```diff
@@ -0,0 +1,109 @@
+Units are organised in systems of units and the global standard for units is the [International System of Units (SI)](https://en.wikipedia.org/wiki/International_System_of_Units).
+Apart from SI there are other systems in use like the imperial unit system. The same physical quantity can be expressed in different units and mathematical transformations exist to convert between them.
+Depending on the users locale or cultural preference a specific unit might fit the users needs. For temperature it is common to use centigrade (°C) in most places around the globe but degree Fahrenheit (°F) is used in the United States of America.
+
+##### UoM data with QuantityType
+Sensor devices and internet services connected to Eclipse SmartHome as Things all provide their scalar values in an implicit unit of measurement. The binding developer knows what exact unit the sensor or service data uses and can now pass those values to the framework using the new [`QuantityType`](../documentation/concepts/units-of-measurement.html). QuantityType represents the scalar value with the corresponding unit. It also serves as the main API to convert between different units.
+The Yahoo Weather binding makes use of QuantityType and publishes state updates in the following form:
+
+```(java)
+QuantityType<Temperature> state = new QuantityType<>(tempDouble, SIUnits.CELSIUS);
+updateState(temperatureChannelUID, state);
+```
+
+For this example the binding developer knows from the Yahoo API description that centigrade is used for temperature values. For pressure values the API description states that Hecto Pascal (hPa) is used, so a state update for the pressure channel looks like this:
+
+```(java)
+QuantityType<Pressure> state = new QuantityType<>(pressDouble, HECTO(SIUnits.PASCAL));
+updateState(pressureChannelUID, state);
+```
+
+In the temperature example the state is marked to be a "<Temperature>" typed QuantitType. It will only accept units which correspond to the dimension type, like centigrade, degree Fahrenheit or Kelvin.
+In the same way the Thing`s Channels and the linked Number items have to be typed to match against each other. Both Channel and Item definition use an extended item type format which passes the expected dimension to a Number item.
+
+The temperature Channel definition for Yahoo Weather looks like this:
+
+```(xml)
+<channel-type id="temperature">
+    <item-type>Number:Temperature</item-type>
+    <label>Temperature</label>
+    <description>Current temperature</description>
+    <category>Temperature</category>
+    <state readOnly="true" pattern="%.1f %unit%"/>
+</channel-type>
+```
+
+A corresponding item definition in the *.items DSL is then:
```

Comment:
I would highlight `*.items`

A corresponding item definition in the `*.items` DSL is ~~then~~: